src/ppom-controller.ts

```diff
@@ -589,35 +584,11 @@ export class PPOMController extends BaseControllerV2<
         ),
       true,
     );
-    if (response?.status !== 200) {
+    const cached = response?.status === 304;
```

Comment:
@jpuri I don't think `fetch` ever actually returns `304` in the browser 🤔 

IIRC it returns the headers for the cached request, so it would likely be `200` even if the request is cached.